.vscode/settings.json

```diff
@@ -2,13 +2,16 @@
   "editor.tabSize": 2,
   "files.eol": "\n",
   "typescript.tsdk": ".yarn/sdks/typescript/lib",
+  "typescript.tsserver.watchOptions": {
+    "watchFile": "useFsEventsOnParentDirectory",
+    "watchDirectory": "useFsEvents"
+  },
```

Comment:
How does Windows deal with this?

Reply:
`FsEvents` is kind of a misnomer, it actually just uses `fs.watch`